addons/Media_Recorder/src_webpack/MediaRecorder.jsm

```diff
@@ -86,7 +90,7 @@ export class MediaRecorder {
     }
 
     getState() {
-        return JSON.stringify(this.addonState);
+        return this.addonState.getState();
```

Comment:
Zmieniłeś format stanu w getState, ale już w setState nie wprowadziłeś żadnych zmian. Zwróć też uwagę na kwestię kompatybilności wstecznej ze starymi stanami.

Reply:
setState korzysta z Object.assign, który to kopiuje i podmienia wartości dla kluczy, które się powtarzają: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign. this.addonState to objekt AddonState, dla którego dodałem nowe property this.mp3ConvertHandler. W zmodyfikowanym getState jest on ignorowany, więc nie powinien nigdy znaleźć się w stanie, tym samym nigdy zostać podmieniony w setState co już by powodowałoby błędy. 

W celach upewnienia mogę napisać testy. Jeżeli nic by nie wykryły to oddałbym do CodeReview bez ponownego wystawiania wersji

Reply:
Został dodany test oraz sprawdzono manualnie czy po wczytaniu starego stanu addon zachowuje się prawidłowo